src/block-components/link/attributes.js

```diff
@@ -3,7 +3,7 @@ import { addLinkAttributes } from '../helpers/link'
 export const addAttributes = ( attrObject, options = {} ) => {
 	const {
 		attrNameTemplate = 'link%s',
-		selector = 'a',
+		selector = 'a.stk-link',
```

Comment:
Instead of changing the default value for everyone that uses the `selector`, can you instead provide `a.stk-link` to the block/component that needs this?
```suggestion
		selector = 'a',
```